apiml-security-common/src/main/java/com/ca/apiml/security/common/config/AuthConfigurationProperties.java

```diff
@@ -62,9 +65,15 @@
         private Integer cookieMaxAge = -1;
     }
 
+    @Data
+    public static class PassTicket {
```

Comment:
What is this class for? I understand it is designed to have ```apiml.security.auth.passTicket.timeout``` property. But it is totally unused. 
I checked ```apiml.security.auth.passTicket.timeout``` param is being used with ```@Value("${apiml.security.auth.passTicket.timeout:540}")```. So this param is called by spring feature, not by ```AuthConfigurationProperties```.
Another point is that default value of apiml.security.auth.passTicket.timeout is mentioned as two value: 
```java
   @Data
    public static class PassTicket {
        private Integer timeout = 360;
    }
```
```java
@Value("${apiml.security.auth.passTicket.timeout:540}")
```